lib/pharos/config_schema.rb

```diff
@@ -56,6 +56,7 @@ def self.messages
               optional(:ssh_key_path).filled
               optional(:container_runtime).filled(included_in?: ['docker', 'cri-o'])
               optional(:http_proxy).filled(:str?)
+              optional(:no_proxy).filled(:str?)
```

Comment:
Just wondering if we should break backwards compatibility and change proxy keys to something like this:
```yaml
proxy:
  http: ""
  no_proxy: ""
```

Reply:
+1 for this change

Reply:
@kke ping

Reply:
Does it make sense to have it per-host?

Reply:
Or should it maybe actually be `environment:`

Reply:
In which case the whole thing could be 1) supplied to the Net::SSH::Start as is to be used in commands and 2) written into `/etc/environment` in a phase for daemons.

Reply:
Generic `environment:` makes sense since these all are actually "just" environment variables. I would still keep these per host, we can later add "global" parameters that can be overridden per host.